2024-12-21 22:46:37 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /4k4UDcF9%2F889478%2F66697832db1b3b589eb846eeaa057ebc715df088%2F6767452fabbedefa47b44f4d953009dd94826f60?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=C2A44DD53EB149C1BB01925E42C73B4D&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734824767&x-oss-signature=WyRDdhtSdRN%2FPXJyyAoDp%2BGpw14p8YHD9J%2Fa%2Fb3K4gY%3D&x-oss-signature-version=OSS2
2024-12-21 22:50:48 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-21 22:50:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-12-21 22:59:46 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /cUs6271J%2F889478%2F65ba1eac9aba5c6614cc4eb39ebc702ef875b6f6%2F67674840f09de8319c1d4f97b10267ca3b0961c4?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=91590FE5E91C43E8959CC44FEB5501FC&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734825552&x-oss-signature=SbqJpgORb9yw%2FjN4e6X2lY0AXuQqRZrZ1F6NO%2Fc9O3o%3D&x-oss-signature-version=OSS2
2024-12-21 23:10:06 - WARNING - pixivdrive.py - 127 - Failed to add salt to page tmp/113838114/113838114_p0.gif, upload original page.
2024-12-21 23:13:51 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /F1c1Gr7Y%2F889478%2F6705d70944b8879f1a244bf8aeae631ba3909054%2F67674b913e03579a48b443cb822a15a889636a65?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=643DC029136645048917437451985FAF&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734826401&x-oss-signature=nTI%2F0nWPlqeGsq5xCBOekjeLToLdhhFGG3FzaQ%2BR9N0%3D&x-oss-signature-version=OSS2
2024-12-21 23:16:34 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-21 23:16:35 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2024-12-21 23:26:50 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /9TS0kiPm%2F889478%2F67666f744a6b56b182ad4cb8a0906fb2abed0164%2F67674e9bd40003b74cb447d2959de7285af4c8de?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=59E4BF1EA9D048A388AED3512C162A10&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734827179&x-oss-signature=FHJDesLVt1UY8T%2F6O8AZqiqwg%2FGtY3kKwY3S0LGAOcE%3D&x-oss-signature-version=OSS2
2024-12-21 23:30:23 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/07/30/10/58/30/110364999_p0.jpg
2024-12-21 23:39:47 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /LEdR6qjE%2F889478%2F6740753a741538c54dbd4439a408611fdf8da8b9%2F676751a4b2f964321ea9416b9eefc4d202288b10?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=F7F32874E535411F9A001F78D7C4810B&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734827956&x-oss-signature=JrrfWuKA3kBMjvgOTgmkNBPdIqrkp72dVwm%2FLHP3KzQ%3D&x-oss-signature-version=OSS2
2024-12-21 23:52:24 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /ECgU9ZYF%2F889478%2F676663594790b5df3ca54eb6819ba3ba95f49474%2F6767549925f58e85c461415aa845a025180af73f?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=D7E416DB37DE4EC09C5060205ED42B92&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734828713&x-oss-signature=KkOXCvRuxgSA%2BP63ULzZHLVOWkPKQuIGk%2FFJJcJc9bo%3D&x-oss-signature-version=OSS2
2024-12-22 00:05:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /2joBT6SB%2F889478%2F6750d73893006c96acc44950a2094460157f8e7e%2F676757ba0057657f30b848b0923bfbbcb099df85?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=AEE79D987818427BB6ECBD2D1D59BC97&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734829514&x-oss-signature=312A%2B2jj2u2h7NiT6h5N1teAdOR4YMi3UG4AT4cR1ZY%3D&x-oss-signature-version=OSS2
2024-12-22 00:11:11 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-22 00:19:30 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /tRsNhlQf%2F889478%2F6766570b1c3471a604ea495b9a8f393d09d441ba%2F67675af4ce1f649a390045ef83ae88d4e2bdf6f9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=904532EE23B94B32A53442F5929B84B7&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734830340&x-oss-signature=4kddimdChrC1cPf3vR1N3JuUgmsKn%2Bc4%2FkmYFrU390M%3D&x-oss-signature-version=OSS2
2024-12-22 00:23:43 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/17/19/28/27/121578673_p0.jpg
2024-12-22 00:31:18 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-22 00:31:19 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2024-12-22 00:34:39 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /LnFQyMGq%2F889478%2F651e8fc7da6a128b39934bb78335c79c9ce0ac53%2F67675e81259d67e62c724e1c9ddb882826d1d2f9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDDdPu1IdM9oqqVh%2BEsjg%2FY9tOq67ghTz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVesTmSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew%2F5X%2FUW17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiinnYT%2FXtuMkagAGKqOHb%2FtBi3lRl%2F660AtRI8sNJEIjL1woyhRE4GAuqHPQWfJSsnfmJrJKA5Y7GejrDnyzO9hmn7n7FdxUheiDAcA6Bsiv82C1maNU0dJxIIFauNrjMP%2Fhf1x6fxwMxVhI%2FotO4Mb0OyuN84Ibmq%2Bh10guX%2B24yCuPtJ%2BrwdMoOkiAA&uploadId=4A23F3956905400A8A2FBB7890C65B13&x-oss-access-key-id=STS.NV6FiZ9XmAHAT95V8joTbDkKg&x-oss-expires=1734831249&x-oss-signature=GBDS1uKcBRPic5n6lEg5kPaUESZG01TDvFou3s401Uw%3D&x-oss-signature-version=OSS2
2024-12-22 05:12:54 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-22 05:51:41 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/11/18/52/45/121393232_p0.png
2024-12-22 06:00:59 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /OQ3wKxJ5%2F889478%2F67011a456c36eea58a6d40ad93b4a2ca26fc6117%2F6767aafd8970c6c35a9e4b93947af08ab99533a1?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5f%2FAfDNh4lS1vSTenD6oG0nOeZ%2B24j5rjz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWOwMWOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA3xSziu17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSimq2IUfXtuMkagAGL9j2K08aufMPuaznu32qwHMEOPU06eI1NFRwJ85ySHNVlJOKoThXPEW%2B6Lzqroc2l3Zxeki7rZcPxTY%2FDBdVffcQbqpz7n0msRo48xq15gvZFV5P1k0HsJXCSboOuzgYlyvQtNKyXua6Io6yhzkdk07QloEQcaG66tXngCa%2F8SyAA&uploadId=9A8A97EAB7B74556B227EC40E149908E&x-oss-access-key-id=STS.NTJJJyjVsa6xxVKDmr5iR4MRL&x-oss-expires=1734850829&x-oss-signature=A%2FN2Hg3ef8rwkkImyZYvJ9z1IicpWRHmshSmYS3E7j0%3D&x-oss-signature-version=OSS2
2024-12-22 06:01:46 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/02/00/01/03/121091464_p0.jpg
2024-12-22 06:05:04 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/04/17/12/05/123990875_p0.jpg
2024-12-22 06:08:41 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/12/16/18/17/15/125244678_p0.png
2024-12-22 06:12:07 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/12/09/20/21/29/125042709_p0.jpg
2024-12-22 06:15:05 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/15/18/09/23/120557655_p0.png
2024-12-22 06:52:03 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/12/07/18/20/40/124976732_p0.jpg
2024-12-22 07:11:55 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-22 07:21:40 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/06/26/17/30/21/119990504_p0.png
2024-12-22 07:27:39 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/10/17/00/41/00/123397511_p0.png
2024-12-22 07:42:37 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /TpicjcmA%2F889478%2F666c30956145eb208a3b4fffb43bcfb8d623d28c%2F6767c2cf763100d4707548d0a71cb08d80929968?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DYOPzOlL5g1PusVlDBpkRsYcdInpLggzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVaDIFqAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5bJzC32i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i8z3tLvXtuMkagAGGumRWAknuxty7mX6RTEtU26lS2vPgGvcjPuhDZbDZZ3WgIEw9MfGm6l2uV57DsoJHuPbAVvD2BXZFWksNGz1BODLiJt7LFseHzIj92FvgC4hRgpVnyHgTX0YjTjuph3xIFh6L1%2Fz%2FMk6TSHYlzs3%2F%2B6vgX5TL8X7cGqyNTtYUQSAA&uploadId=FB77CF7296764943A90DE44AC91C5F6B&x-oss-access-key-id=STS.NSmsFzyaAc9GTvpBD9mHdqWKa&x-oss-expires=1734856927&x-oss-signature=DuXxkCAfkxMuIo%2BVC7s6Sgpuxurtz4i4M8Jvgos23lg%3D&x-oss-signature-version=OSS2
2024-12-22 07:49:58 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/BIkCth2L%2F889478%2F66eeb2401b537742eae54244b4e492afcd577a8e%2F6767c4a60de29c87120841c18c23ce4a74919764?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DYOPzOlL5g1PusVlDBpkRsYcdInpLggzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVaDIFqAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5bJzC32i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i8z3tLvXtuMkagAGGumRWAknuxty7mX6RTEtU26lS2vPgGvcjPuhDZbDZZ3WgIEw9MfGm6l2uV57DsoJHuPbAVvD2BXZFWksNGz1BODLiJt7LFseHzIj92FvgC4hRgpVnyHgTX0YjTjuph3xIFh6L1%2Fz%2FMk6TSHYlzs3%2F%2B6vgX5TL8X7cGqyNTtYUQSAA&uploadId=E52C8C776B974FA9A481F9A4D70E0F0E&x-oss-access-key-id=STS.NSmsFzyaAc9GTvpBD9mHdqWKa&x-oss-expires=1734857398&x-oss-signature=smYMAabUgcYrsNjZtIuDFLBv7Q3ByX3edJrRdWtCPC0%3D&x-oss-signature-version=OSS2:502:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>ServiceUnavailable</Code>
  <Message>Sorry, the page you are looking for is currently unavailable. Please try again later.</Message>
  <RequestId>6767C4A6415F1638370F2758</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
</Error>

2024-12-22 22:12:15 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-22 23:08:27 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/09/19/15/24/124135822_p0.jpg
2024-12-22 23:08:39 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/05/07/13/49/00/118516921_p0.jpg
2024-12-22 23:56:13 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/09/10/19/00/08/111610491_p0.jpg
2024-12-23 00:02:48 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/09/16/22/01/55/122498645_p0.jpg
2024-12-23 00:04:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/16/00/01/06/120569961_p0.png
2024-12-23 00:09:38 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/11/27/18/16/20/113758089_p0.jpg
2024-12-23 00:11:16 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-23 05:14:17 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-23 05:16:30 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /ts7sb369%2F889478%2F6768f210b5aa0d834609490cb3fee90e4c8c0cdb%2F6768f2101703363116b84abd83918ba262255a5e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=3A7CB54DF2334B97ADF0A8A2C5B261EF&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734934560&x-oss-signature=HX%2BZNg%2BUpVZX%2FZhIUbapP8oYT%2FEvXkwM9GuhBhdIK60%3D&x-oss-signature-version=OSS2
2024-12-23 05:25:13 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /IhLlbLGi%2F889478%2F6767a3a1bf7ffcfccc3e479d8a481d6c68611661%2F6768f409f91c4ffb96e3488e865afdfd8a44d91b?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=F5EB63B8EFC542108CB5093E076467A2&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935065&x-oss-signature=Ozo958tC4ncDCFeXAL7JF5tyASFGKmYwvYH7p4wq0wo%3D&x-oss-signature-version=OSS2
2024-12-23 05:25:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /IhLlbLGi%2F889478%2F6767a3a1bf7ffcfccc3e479d8a481d6c68611661%2F6768f409f91c4ffb96e3488e865afdfd8a44d91b?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=F5EB63B8EFC542108CB5093E076467A2&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935065&x-oss-signature=Ozo958tC4ncDCFeXAL7JF5tyASFGKmYwvYH7p4wq0wo%3D&x-oss-signature-version=OSS2
2024-12-23 05:28:42 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /Na6Tbq4n%2F889478%2F673bc2958b2d1db50a984fcc889b23cf1e3de389%2F6768f4eccf6a3ac89edf4d1abbe587323fd6cb08?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9718F514789E42EEBED40A03D2CC5177&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935292&x-oss-signature=5oFvtXk9Q7RYZvMs3ubW9bmube1oecRLzTdYu2RdP5w%3D&x-oss-signature-version=OSS2
2024-12-23 05:28:45 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/Na6Tbq4n%2F889478%2F673bc2958b2d1db50a984fcc889b23cf1e3de389%2F6768f4eccf6a3ac89edf4d1abbe587323fd6cb08?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9718F514789E42EEBED40A03D2CC5177&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935292&x-oss-signature=5oFvtXk9Q7RYZvMs3ubW9bmube1oecRLzTdYu2RdP5w%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6768F50B31A03438357372B6</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"E9735B8F72F30E9480A89A3E1EACD129"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-23 05:28:46 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/Na6Tbq4n%2F889478%2F673bc2958b2d1db50a984fcc889b23cf1e3de389%2F6768f4eccf6a3ac89edf4d1abbe587323fd6cb08?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9718F514789E42EEBED40A03D2CC5177&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935292&x-oss-signature=5oFvtXk9Q7RYZvMs3ubW9bmube1oecRLzTdYu2RdP5w%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6768F50D77A7CA34395FE034</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"E9735B8F72F30E9480A89A3E1EACD129"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-23 05:28:46 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/Na6Tbq4n%2F889478%2F673bc2958b2d1db50a984fcc889b23cf1e3de389%2F6768f4eccf6a3ac89edf4d1abbe587323fd6cb08?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9718F514789E42EEBED40A03D2CC5177&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935292&x-oss-signature=5oFvtXk9Q7RYZvMs3ubW9bmube1oecRLzTdYu2RdP5w%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6768F50E26E313383314A6EC</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"E9735B8F72F30E9480A89A3E1EACD129"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-23 05:28:46 - ERROR - aliyundrive.py - 1077 - File tmp/124275635/124275635_p0.jpg Part 1 upload failed.
2024-12-23 05:37:08 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /FdHskhTU%2F889478%2F67568a40481d30d822aa41d1861ecf773cf984f8%2F6768f6e677b670589eba4919a06aa5339efdae25?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=09E6478FAAFF435ABCFD60060B1C9D07&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935798&x-oss-signature=AORMpuVbtg6vvz5b9HMuHnzAW%2BoDQAtsIeXK%2BC%2FsGn0%3D&x-oss-signature-version=OSS2
2024-12-23 05:37:39 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /FdHskhTU%2F889478%2F67568a40481d30d822aa41d1861ecf773cf984f8%2F6768f6e677b670589eba4919a06aa5339efdae25?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=09E6478FAAFF435ABCFD60060B1C9D07&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935798&x-oss-signature=AORMpuVbtg6vvz5b9HMuHnzAW%2BoDQAtsIeXK%2BC%2FsGn0%3D&x-oss-signature-version=OSS2
2024-12-23 05:39:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /tWpC7q5V%2F889478%2F6768f78258d7a6b54f014515851af180c779c2d1%2F6768f782e08e535ccac942458a180784dac266f2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=C67A01958BBE45559A9DB8E754C7BC86&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734935954&x-oss-signature=0GV1SzMpCzYOi3jZuGRkuzqmqHBI09PoYf0yznTjRzk%3D&x-oss-signature-version=OSS2
2024-12-23 05:54:35 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /xqEnqcMN%2F889478%2F67392e37a655674ecf7345a289c843803af42ae6%2F6768fafdebffbd22ee994d0fa468e4d31d0dcf88?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=512B1B3E3A2545E79132D6CDF6B04562&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734936845&x-oss-signature=tjyYYpb%2BJICp5PqkggGO3bt%2BxWWI%2BmjVdZzGZBqOBsk%3D&x-oss-signature-version=OSS2
2024-12-23 05:55:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/05/00/09/40/120243190_p0.jpg
2024-12-23 05:57:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/10/20/00/03/55/123492249_p0.jpg
2024-12-23 05:59:56 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-23 05:59:57 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2024-12-23 06:04:20 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /rcZgccoc%2F889478%2F66f494f7922b883cbe214fdbb10c06eaa67ef549%2F6768fd30ba43d03a2b594fa187ca355a266eeff7?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9AB49C6DCB9A4C29B97740D1F5C93132&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734937408&x-oss-signature=OU%2FRnuIc6865EGnwXDNJa7ZizFLxgg4KgN8YdIJs26Y%3D&x-oss-signature-version=OSS2
2024-12-23 06:04:51 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /rcZgccoc%2F889478%2F66f494f7922b883cbe214fdbb10c06eaa67ef549%2F6768fd30ba43d03a2b594fa187ca355a266eeff7?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9AB49C6DCB9A4C29B97740D1F5C93132&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734937408&x-oss-signature=OU%2FRnuIc6865EGnwXDNJa7ZizFLxgg4KgN8YdIJs26Y%3D&x-oss-signature-version=OSS2
2024-12-23 06:05:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /rcZgccoc%2F889478%2F66f494f7922b883cbe214fdbb10c06eaa67ef549%2F6768fd30ba43d03a2b594fa187ca355a266eeff7?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9AB49C6DCB9A4C29B97740D1F5C93132&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734937408&x-oss-signature=OU%2FRnuIc6865EGnwXDNJa7ZizFLxgg4KgN8YdIJs26Y%3D&x-oss-signature-version=OSS2
2024-12-23 06:05:53 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/rcZgccoc%2F889478%2F66f494f7922b883cbe214fdbb10c06eaa67ef549%2F6768fd30ba43d03a2b594fa187ca355a266eeff7?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9AB49C6DCB9A4C29B97740D1F5C93132&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734937408&x-oss-signature=OU%2FRnuIc6865EGnwXDNJa7ZizFLxgg4KgN8YdIJs26Y%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /rcZgccoc%2F889478%2F66f494f7922b883cbe214fdbb10c06eaa67ef549%2F6768fd30ba43d03a2b594fa187ca355a266eeff7?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=9AB49C6DCB9A4C29B97740D1F5C93132&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734937408&x-oss-signature=OU%2FRnuIc6865EGnwXDNJa7ZizFLxgg4KgN8YdIJs26Y%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-23 06:08:32 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /fJ5mVW2D%2F889478%2F6768fe427e7d20f2db7141e48923643dd2233d0e%2F6768fe426b00601b95204e5f913bdfa5561d5672?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=CA011548EC80457FB3675AC413ECD742&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734937682&x-oss-signature=ksMk1ki%2Bv0w%2BqqACtEd67RVRaDkLTUeHRfcANCjuemk%3D&x-oss-signature-version=OSS2
2024-12-23 06:09:34 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/18/21/39/27/124425092_p0.jpg
2024-12-23 06:22:19 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /wh3WXN8j%2F889478%2F66b7524a99e613c48a0747809a5d5fd69beae074%2F6769017d4ab3eede3583451b9e68664cd3ebcfd9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=AADEA63C06564C0AB31AA3453E78AFAD&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734938509&x-oss-signature=5suUtXUSBg84shrBCkdEWYUhn36bQK0geJ1oxw%2F2z0s%3D&x-oss-signature-version=OSS2
2024-12-23 06:29:56 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /lWCnYLiC%2F889478%2F6756c2c62062c7ea4e32442a8bab61ce3b4d57d1%2F67690346647981f581d047a4993eb825736b66ac?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=BA8F6660738448AA989D5334E26E85B7&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734938966&x-oss-signature=D9pY24Yj9J%2FKe3O5yyYY0xz0k%2FdVV24aOQi674kD3XU%3D&x-oss-signature-version=OSS2
2024-12-23 06:30:27 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /lWCnYLiC%2F889478%2F6756c2c62062c7ea4e32442a8bab61ce3b4d57d1%2F67690346647981f581d047a4993eb825736b66ac?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=BA8F6660738448AA989D5334E26E85B7&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734938966&x-oss-signature=D9pY24Yj9J%2FKe3O5yyYY0xz0k%2FdVV24aOQi674kD3XU%3D&x-oss-signature-version=OSS2
2024-12-23 06:31:19 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /eZ6KwVob%2F889478%2F6756c30b1cc4268e273f4e4e843b50e39abd796e%2F67690399d0583d2ede4f4128af9916fbbeb39e41?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=DF91F7FDD9164FA8B638D22A37E1A90F&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734939049&x-oss-signature=UqdfFRggDcVapJV50zQnL9kHLAaCQ%2BMUE8lfNqUkx8Y%3D&x-oss-signature-version=OSS2
2024-12-23 06:31:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /eZ6KwVob%2F889478%2F6756c30b1cc4268e273f4e4e843b50e39abd796e%2F67690399d0583d2ede4f4128af9916fbbeb39e41?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bxJYiCneh24vCldHLho2I0QdtKmfTfoDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNRr4VjSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYne4zVReT%2B2i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16i3TWXXvXtuMkagAE409K7f6m6bFMIlStI6Qh0usziFERK%2BKRd3exLfU13vYTALY5RSFzQidjoQ8N5oRG7GGtJ7P9byYF5H5EkMso4FZu3CZWyK8XRXFKszMv23Wl96VU2yXKM725RGXKrIZglinNZasmIvx3ZhifkItpOUglS7F3Kys%2FwPPmpd2ImdyAA&uploadId=DF91F7FDD9164FA8B638D22A37E1A90F&x-oss-access-key-id=STS.NUDn26p7WU2NvTPGbaMTfv1tB&x-oss-expires=1734939049&x-oss-signature=QT0O%2FJTi7l5z%2FKMQhEuJH7meiNnHAtUSkl5v1iQtwMI%3D&x-oss-signature-version=OSS2
2024-12-23 06:34:43 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /O4MX7I1L%2F889478%2F6756c376d4640fdc579145fcb420d9836b233909%2F67690465bdaefa2b2a9a44cf8dd2459524a11b3d?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=7D4ABEA18E0446C1999E8FAA9E1502F7&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734939253&x-oss-signature=gl4%2FOkvFuoqWJy75I9etDsqUMALH83Rl69DhdE0X02k%3D&x-oss-signature-version=OSS2
2024-12-23 06:37:06 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/10/11/00/00/17/123214277_p0.jpg
2024-12-23 06:45:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /idBR7UMK%2F889478%2F664fc4d1dd0d5eeba94b47f6b1e75adccf0be640%2F676906e398c5cc0c9be44c07873713f9f14dc44e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=C2C165C464274A0B81F81FAC9E4E52BE&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734939891&x-oss-signature=6l2hJlFTaVV0qU7Klw81vp709osUg348RCF0au9ZWHc%3D&x-oss-signature-version=OSS2
2024-12-23 06:45:52 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /idBR7UMK%2F889478%2F664fc4d1dd0d5eeba94b47f6b1e75adccf0be640%2F676906e398c5cc0c9be44c07873713f9f14dc44e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=C2C165C464274A0B81F81FAC9E4E52BE&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734939891&x-oss-signature=6l2hJlFTaVV0qU7Klw81vp709osUg348RCF0au9ZWHc%3D&x-oss-signature-version=OSS2
2024-12-23 06:46:23 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /idBR7UMK%2F889478%2F664fc4d1dd0d5eeba94b47f6b1e75adccf0be640%2F676906e398c5cc0c9be44c07873713f9f14dc44e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=C2C165C464274A0B81F81FAC9E4E52BE&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734939891&x-oss-signature=6l2hJlFTaVV0qU7Klw81vp709osUg348RCF0au9ZWHc%3D&x-oss-signature-version=OSS2
2024-12-23 06:49:33 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /6rDiIhcB%2F889478%2F676907de319f18f7addb408da53aa67ba11e77cf%2F676907de5f2038e2b8c5423fb4a23b053afc1cc9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=51AED04383F54FF1857DF70D58BF668B&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734940142&x-oss-signature=glaENEZ5lQVSy%2B4l9NT8mFht9gd6gbfudSu0fAJyaTo%3D&x-oss-signature-version=OSS2
2024-12-23 07:06:57 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /fdZmq2Us%2F889478%2F64125f87eebc9dbeff4146a98c9eba4bf42e2cde%2F67690be2efa405aba544464aa100ec4d48c849c9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=C7CF1193741A4CF99B1728EDD871B270&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734941171&x-oss-signature=Z074A7tm4i2cK9EJI1dgIvLmMQ6qv1%2FAO5A345Stvh8%3D&x-oss-signature-version=OSS2
2024-12-23 07:12:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/05/29/15/56/44/119155540_p0.jpg
2024-12-23 07:12:57 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/06/08/11/17/42/108826801_p0.jpg
2024-12-23 07:13:18 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-23 07:23:04 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /QJUZmZB1%2F889478%2F670cf1d327c4068350a44fff9497e39765bb51dc%2F67690fbaeae998192cae478b91b928abd3769f68?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=8732260D3A97467C81949AB9A81D5FFC&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734942154&x-oss-signature=ZBLQGw8XIeqzpBQiBF1A6ekAkTdW1soMzzYWUabF8T0%3D&x-oss-signature-version=OSS2
2024-12-23 07:36:54 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /bP9lIKAK%2F889478%2F66f4a08b1662cac62a8d4dbbb0b8fadd345feb43%2F676912e1a6f2f8a89bec4a878f4b21d05bcece03?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=E73CE3EDE85F4D8C90395A5FC6EFA9F9&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734942961&x-oss-signature=GBeyXANUnIxIa2lDbkKEQGjXrkK7bhbufIC4Bpmtbm0%3D&x-oss-signature-version=OSS2
2024-12-23 07:36:58 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/bP9lIKAK%2F889478%2F66f4a08b1662cac62a8d4dbbb0b8fadd345feb43%2F676912e1a6f2f8a89bec4a878f4b21d05bcece03?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=E73CE3EDE85F4D8C90395A5FC6EFA9F9&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734942961&x-oss-signature=GBeyXANUnIxIa2lDbkKEQGjXrkK7bhbufIC4Bpmtbm0%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6769131790232A3133041E29</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"1CA72B5322D87CDF70F7F4E88759ED18"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-23 07:36:58 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/bP9lIKAK%2F889478%2F66f4a08b1662cac62a8d4dbbb0b8fadd345feb43%2F676912e1a6f2f8a89bec4a878f4b21d05bcece03?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=E73CE3EDE85F4D8C90395A5FC6EFA9F9&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734942961&x-oss-signature=GBeyXANUnIxIa2lDbkKEQGjXrkK7bhbufIC4Bpmtbm0%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6769131AECF8C938349D13AA</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"1CA72B5322D87CDF70F7F4E88759ED18"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-23 07:37:01 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/bP9lIKAK%2F889478%2F66f4a08b1662cac62a8d4dbbb0b8fadd345feb43%2F676912e1a6f2f8a89bec4a878f4b21d05bcece03?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=E73CE3EDE85F4D8C90395A5FC6EFA9F9&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734942961&x-oss-signature=GBeyXANUnIxIa2lDbkKEQGjXrkK7bhbufIC4Bpmtbm0%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6769131A4277683936A880AE</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"1CA72B5322D87CDF70F7F4E88759ED18"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-23 07:37:01 - ERROR - aliyundrive.py - 1077 - File tmp/114981509_mirage/114981509_p0.png Part 1 upload failed.
2024-12-23 07:52:14 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /7BXgR4qO%2F889478%2F66f4a449fdecf8e01d4548a69305d38c95b93369%2F6769167fa5dca71e0e764e1ea747d4a759cc87d3?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=6BD93D9B73394609B09FE2AFA34A8340&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734943887&x-oss-signature=5Y2jwNT6OxH3UXbNaLXD22rCNbywKzncE2PgpuhQBRo%3D&x-oss-signature-version=OSS2
2024-12-23 07:57:32 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/09/29/20/30/04/122884024_p0.png
2024-12-23 08:00:14 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/14/14/05/20/121477946_p0.jpg
2024-12-23 08:21:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /ZyBTw5aO%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F67691d6bc177ac573f814e2ea2eab593ad013a54?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=A5FD44E119E94B45846FAF3C18FB6FC1&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734945659&x-oss-signature=STby8G%2B3zwZKBFs15%2Fzgk2bZ%2B2nnYCanlUP2%2BgNYxzQ%3D&x-oss-signature-version=OSS2
2024-12-23 08:32:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /AkJ5dkkn%2F889478%2F657ecd3e1af932b94b9a4c4fb2560520658cbc09%2F6769201b2ba16ef5e7934dc4a0e4f6cea23c89a5?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5fAD4%2F4lZN0joiJWk3gjUYZW%2Bgejp3FrDz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUnLOTOAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ%2FlIyiw17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSivLs73fXtuMkagAGO1BBGF0NXU1udpdAPZO0oi%2BV21JhpOM8F0DzUdgf6%2BufGxlmY3y75gZY7%2FQwenti7dqaHxjmKb1T7QoyB%2B6HtFWu6mE0ly88LleRolPbbVkJQ%2F5p7ZCwca%2FfZenWZf1Hst0wzCreYZ2u%2B1EYigf%2Bd%2FoxsdNMTVemsqtViepU77CAA&uploadId=067BA62F70534CE2A80CEC3787A342FA&x-oss-access-key-id=STS.NTuD5LxLU9JbXkQiFLWg2aXnN&x-oss-expires=1734946347&x-oss-signature=78PR3oyAsVZk0rMz30Z7J4wVd1OGXw8o%2FxQcRIqzFkI%3D&x-oss-signature-version=OSS2
2024-12-23 08:51:23 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/26/22/25/31/121862084_p0.jpg
2024-12-23 22:13:08 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-23 22:45:11 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/10/05/00/40/09/123038065_p0.jpg
2024-12-23 23:12:30 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/05/15/00/06/21/98348219_p0.png
2024-12-23 23:38:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/01/00/04/44/120132020_p0.jpg
2024-12-24 05:13:52 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-24 05:14:56 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /pbPkD12g%2F889478%2F67593b1b29c92bbc7d7a4125b62a03cd719884da%2F676a43321a1cbb3ac2464f7580c959f11b5b3159?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=54F43FD0731B4B4285A709466A0F6F94&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735020866&x-oss-signature=mYrOGPy8o%2FoF1rs77MNPBt7ffrlPHHJZUw%2B%2Ba55JdM0%3D&x-oss-signature-version=OSS2
2024-12-24 05:22:24 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2
2024-12-24 05:22:55 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2
2024-12-24 05:23:14 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A45303A74A6373984A3D9</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"10AD6622DDDA3EEC1C84DA4FEF35C234"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 05:23:39 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A45428CFC8233373B2303</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"10AD6622DDDA3EEC1C84DA4FEF35C234"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 05:24:09 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2
2024-12-24 05:24:40 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2
2024-12-24 05:25:10 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2
2024-12-24 05:25:41 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /inPiSNqd%2F889478%2F672c0cc7a9f00416d592409e872ef6dd7a759e5d%2F676a44f234fb4892db86465ebe472b1286542fbc?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=DF75A20BB03C443A8ED99E4DAE5E3DA9&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021314&x-oss-signature=b5yuEjJ71Dw4u%2Bf217X1p%2FqXqNGqKxA7Id8siEg02hw%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 05:25:41 - ERROR - aliyundrive.py - 1077 - File tmp/122995407/122995407_p0.png Part 1 upload failed.
2024-12-24 05:32:33 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /hjmEi27k%2F889478%2F65e061e4c5c602e30fa84808a97d239933033c3c%2F676a4752c373259f566140cc9fdae2a77f183f8f?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F2D6429F89364D1BB1B156D5C005CA2F&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021922&x-oss-signature=4kmKPkm7PTO8uW%2BDadYotisrRg4ui5YX2bXinD3NqKc%3D&x-oss-signature-version=OSS2
2024-12-24 05:33:04 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /hjmEi27k%2F889478%2F65e061e4c5c602e30fa84808a97d239933033c3c%2F676a4752c373259f566140cc9fdae2a77f183f8f?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F2D6429F89364D1BB1B156D5C005CA2F&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021922&x-oss-signature=4kmKPkm7PTO8uW%2BDadYotisrRg4ui5YX2bXinD3NqKc%3D&x-oss-signature-version=OSS2
2024-12-24 05:33:34 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /hjmEi27k%2F889478%2F65e061e4c5c602e30fa84808a97d239933033c3c%2F676a4752c373259f566140cc9fdae2a77f183f8f?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F2D6429F89364D1BB1B156D5C005CA2F&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021922&x-oss-signature=4kmKPkm7PTO8uW%2BDadYotisrRg4ui5YX2bXinD3NqKc%3D&x-oss-signature-version=OSS2
2024-12-24 05:34:06 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/hjmEi27k%2F889478%2F65e061e4c5c602e30fa84808a97d239933033c3c%2F676a4752c373259f566140cc9fdae2a77f183f8f?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F2D6429F89364D1BB1B156D5C005CA2F&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021922&x-oss-signature=4kmKPkm7PTO8uW%2BDadYotisrRg4ui5YX2bXinD3NqKc%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /hjmEi27k%2F889478%2F65e061e4c5c602e30fa84808a97d239933033c3c%2F676a4752c373259f566140cc9fdae2a77f183f8f?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F2D6429F89364D1BB1B156D5C005CA2F&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735021922&x-oss-signature=4kmKPkm7PTO8uW%2BDadYotisrRg4ui5YX2bXinD3NqKc%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 05:34:51 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2
2024-12-24 05:35:23 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2
2024-12-24 05:35:53 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2
2024-12-24 05:36:24 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 05:36:27 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A4859CBC22F3032A093CE</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"B81F620C2A920AF6E7B4EBB028BB29F9"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 05:36:28 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/gMxLy7YN%2F889478%2F65e061e6cf629e53500949d9b1de944abdeefab2%2F676a47d347ee963a05f544318252b454354baccd?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=397133DFE1A44B308E7B8A2CDC81276D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022051&x-oss-signature=%2FDPjaer8KEndbehc0WEJ%2BN7du5O5pGqH2yL5TkfAtvg%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A485B6CD79F3836D4B92C</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"B81F620C2A920AF6E7B4EBB028BB29F9"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 05:36:28 - ERROR - aliyundrive.py - 1077 - File tmp/116438737/116438737_p20.png Part 1 upload failed.
2024-12-24 05:42:53 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /QE78d2Kd%2F889478%2F6756aaff5853d44ace7142c1af8b72e2e2dff4d9%2F676a49bf17d190e945af456ba78a77a818ef4c66?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=892CA1617B464502BDBE4CDB7A3C67A7&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022543&x-oss-signature=cM69EzmD3qimnOAy%2BSMG3hUSGLL74oTN5QIZsb6gLIk%3D&x-oss-signature-version=OSS2
2024-12-24 05:43:25 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /QE78d2Kd%2F889478%2F6756aaff5853d44ace7142c1af8b72e2e2dff4d9%2F676a49bf17d190e945af456ba78a77a818ef4c66?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=892CA1617B464502BDBE4CDB7A3C67A7&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022543&x-oss-signature=cM69EzmD3qimnOAy%2BSMG3hUSGLL74oTN5QIZsb6gLIk%3D&x-oss-signature-version=OSS2
2024-12-24 05:43:56 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /QE78d2Kd%2F889478%2F6756aaff5853d44ace7142c1af8b72e2e2dff4d9%2F676a49bf17d190e945af456ba78a77a818ef4c66?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=892CA1617B464502BDBE4CDB7A3C67A7&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022543&x-oss-signature=cM69EzmD3qimnOAy%2BSMG3hUSGLL74oTN5QIZsb6gLIk%3D&x-oss-signature-version=OSS2
2024-12-24 05:45:18 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2
2024-12-24 05:45:49 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2
2024-12-24 05:46:20 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2
2024-12-24 05:46:51 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 05:46:54 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A4ACCAAA18C3133D890DE</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"580F53CA2BE9E5937DC0C293B107EAA5"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 05:46:54 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/1gCRzvNt%2F889478%2F6747a63d244371e636f9464f89f79b79cd5b5276%2F676a4a37cc44b6d8072b4ba38efa615420a7e443?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=5721F170BEA74F38B5CC211A512D4824&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735022663&x-oss-signature=22SUD5IOl9%2FXniTt8dA2SyWM35xIhDU7ztwyDGVKa5A%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A4ACEB8F8773833887C7F</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"580F53CA2BE9E5937DC0C293B107EAA5"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 05:46:54 - ERROR - aliyundrive.py - 1077 - File tmp/124681623/124681623_p0.png Part 1 upload failed.
2024-12-24 05:53:15 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /lMkSb6Nm%2F889478%2F666d832e0a4d9ebda0e4468eaffffa79f6723060%2F676a4c2ca411dfad98ef495baa5b6333db207877?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E16D9133EF894C178E6F7101232213DD&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023164&x-oss-signature=qonn2R2egGEpRhpKMFk0Py87vMlIt%2FDbVTYTjQxJlXU%3D&x-oss-signature-version=OSS2
2024-12-24 05:53:46 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /lMkSb6Nm%2F889478%2F666d832e0a4d9ebda0e4468eaffffa79f6723060%2F676a4c2ca411dfad98ef495baa5b6333db207877?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E16D9133EF894C178E6F7101232213DD&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023164&x-oss-signature=qonn2R2egGEpRhpKMFk0Py87vMlIt%2FDbVTYTjQxJlXU%3D&x-oss-signature-version=OSS2
2024-12-24 05:55:40 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /fBoYIv0Z%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F676a4cbeaa808f62087e4b578cf1d01443776fb2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=C743DD61B1134D31805A0AE2D291380D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023310&x-oss-signature=4dmYXX9i8QFfAo3IQ4oKkKpDKx1L1E0%2BBfyhVewdx54%3D&x-oss-signature-version=OSS2
2024-12-24 05:56:11 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /fBoYIv0Z%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F676a4cbeaa808f62087e4b578cf1d01443776fb2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=C743DD61B1134D31805A0AE2D291380D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023310&x-oss-signature=4dmYXX9i8QFfAo3IQ4oKkKpDKx1L1E0%2BBfyhVewdx54%3D&x-oss-signature-version=OSS2
2024-12-24 05:56:42 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /fBoYIv0Z%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F676a4cbeaa808f62087e4b578cf1d01443776fb2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=C743DD61B1134D31805A0AE2D291380D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023310&x-oss-signature=4dmYXX9i8QFfAo3IQ4oKkKpDKx1L1E0%2BBfyhVewdx54%3D&x-oss-signature-version=OSS2
2024-12-24 05:57:13 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/fBoYIv0Z%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F676a4cbeaa808f62087e4b578cf1d01443776fb2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=C743DD61B1134D31805A0AE2D291380D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023310&x-oss-signature=4dmYXX9i8QFfAo3IQ4oKkKpDKx1L1E0%2BBfyhVewdx54%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /fBoYIv0Z%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F676a4cbeaa808f62087e4b578cf1d01443776fb2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=C743DD61B1134D31805A0AE2D291380D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023310&x-oss-signature=4dmYXX9i8QFfAo3IQ4oKkKpDKx1L1E0%2BBfyhVewdx54%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 05:57:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /fBoYIv0Z%2F889478%2F67691d6bd921873a536643149b4121812c95a865%2F676a4cbeaa808f62087e4b578cf1d01443776fb2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=C743DD61B1134D31805A0AE2D291380D&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023310&x-oss-signature=4dmYXX9i8QFfAo3IQ4oKkKpDKx1L1E0%2BBfyhVewdx54%3D&x-oss-signature-version=OSS2
2024-12-24 06:02:57 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-24 06:02:58 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2024-12-24 06:04:41 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /M4eNCYS6%2F889478%2F676a4edbe4cab217eee94f4e86eecf96bbf810b5%2F676a4edba1e120a03a2f4d04a4e0500cab548ab5?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=ED20640167DE47B1AF64747B0E91FC7A&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735023851&x-oss-signature=fHVA2rh1Yg5oSJTvm6Ql4szQY266ZD9uQzrLhQR9enY%3D&x-oss-signature-version=OSS2
2024-12-24 06:05:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/03/30/00/01/01/117365490_p0.jpg
2024-12-24 06:07:14 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Ify7i3Kx%2F889478%2F675f68e1e8aa46f8bdc64ee8872b796ff066eddd%2F676a4f74be5e334e64584009af224a21e3540946?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F19B6F11C5CB40B09921AE8351D548A3&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024004&x-oss-signature=YDS9g%2Fqfj8lH%2Blbvr4fUcrZ0Q6Gh2WnmJUz3oFMj7Mw%3D&x-oss-signature-version=OSS2
2024-12-24 06:07:45 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Ify7i3Kx%2F889478%2F675f68e1e8aa46f8bdc64ee8872b796ff066eddd%2F676a4f74be5e334e64584009af224a21e3540946?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F19B6F11C5CB40B09921AE8351D548A3&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024004&x-oss-signature=YDS9g%2Fqfj8lH%2Blbvr4fUcrZ0Q6Gh2WnmJUz3oFMj7Mw%3D&x-oss-signature-version=OSS2
2024-12-24 06:08:16 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Ify7i3Kx%2F889478%2F675f68e1e8aa46f8bdc64ee8872b796ff066eddd%2F676a4f74be5e334e64584009af224a21e3540946?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F19B6F11C5CB40B09921AE8351D548A3&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024004&x-oss-signature=YDS9g%2Fqfj8lH%2Blbvr4fUcrZ0Q6Gh2WnmJUz3oFMj7Mw%3D&x-oss-signature-version=OSS2
2024-12-24 06:08:47 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/Ify7i3Kx%2F889478%2F675f68e1e8aa46f8bdc64ee8872b796ff066eddd%2F676a4f74be5e334e64584009af224a21e3540946?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F19B6F11C5CB40B09921AE8351D548A3&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024004&x-oss-signature=YDS9g%2Fqfj8lH%2Blbvr4fUcrZ0Q6Gh2WnmJUz3oFMj7Mw%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /Ify7i3Kx%2F889478%2F675f68e1e8aa46f8bdc64ee8872b796ff066eddd%2F676a4f74be5e334e64584009af224a21e3540946?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=F19B6F11C5CB40B09921AE8351D548A3&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024004&x-oss-signature=YDS9g%2Fqfj8lH%2Blbvr4fUcrZ0Q6Gh2WnmJUz3oFMj7Mw%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 06:16:53 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /qMqsdcua%2F889478%2F676a51b7d1f8856e6df34f71acabca0a760e2ed7%2F676a51b76b687c274a254577bf1c5a8ecdfab3b3?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=4BC6AF1DC003463F8EE4DB90C6416D53&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024583&x-oss-signature=SNr8CMX8rrADa7zzL7nlAwrOvEuR5kJZYL%2FtOzfYKxM%3D&x-oss-signature-version=OSS2
2024-12-24 06:17:25 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /qMqsdcua%2F889478%2F676a51b7d1f8856e6df34f71acabca0a760e2ed7%2F676a51b76b687c274a254577bf1c5a8ecdfab3b3?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=4BC6AF1DC003463F8EE4DB90C6416D53&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024583&x-oss-signature=SNr8CMX8rrADa7zzL7nlAwrOvEuR5kJZYL%2FtOzfYKxM%3D&x-oss-signature-version=OSS2
2024-12-24 06:18:10 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /yvCKweNs%2F889478%2F674ae7df7c94b6a87ae243ec9583b172bbb88387%2F676a5204387877eeffa144a0b1e2c991f2c4877e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=8FD081FC1C6048639C48DE77C1D6BCAF&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024660&x-oss-signature=F9yBEIF3NEjbn6Fk%2Bc7ay8MoQ%2F6vfNqTb5Ux9IDqF%2Bg%3D&x-oss-signature-version=OSS2
2024-12-24 06:18:41 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /yvCKweNs%2F889478%2F674ae7df7c94b6a87ae243ec9583b172bbb88387%2F676a5204387877eeffa144a0b1e2c991f2c4877e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=8FD081FC1C6048639C48DE77C1D6BCAF&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024660&x-oss-signature=F9yBEIF3NEjbn6Fk%2Bc7ay8MoQ%2F6vfNqTb5Ux9IDqF%2Bg%3D&x-oss-signature-version=OSS2
2024-12-24 06:19:12 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /yvCKweNs%2F889478%2F674ae7df7c94b6a87ae243ec9583b172bbb88387%2F676a5204387877eeffa144a0b1e2c991f2c4877e?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=8FD081FC1C6048639C48DE77C1D6BCAF&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735024660&x-oss-signature=F9yBEIF3NEjbn6Fk%2Bc7ay8MoQ%2F6vfNqTb5Ux9IDqF%2Bg%3D&x-oss-signature-version=OSS2
2024-12-24 06:20:57 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/21/23/10/29/120748307_p0.png
2024-12-24 06:26:12 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2
2024-12-24 06:26:43 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2
2024-12-24 06:27:13 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2
2024-12-24 06:27:44 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 06:28:15 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2
2024-12-24 06:28:46 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Pnf6O3ee%2F889478%2F676a43fbb10de1ecde054026b972550893b30aa4%2F676a53e68703efde35ee4e4db27035bf018e80c2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DaJdTDmpFL%2FaGuMW6GokQQe%2BFGoaDTtzz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNVCPbA2Aq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYnew5Brno13ObP7AhWvDNQ3S7jN6YihvSt3zmA4YsrdqJPW1dKDogPIx4aBwHbHMFKlwddMkwuiJsW8gfXtuMkagAEyIa2uCyLogGSmUh2LjXLBpLNZkqJmkOJQ2N4GkJsJ7zrufVaVr7ThW49nAhM2H6OepCAzVKXxggh6uYe6OKqPyKYNfX%2BzfOcRSnN8QF7T6mb0ba3zzhOfFkyf%2F53CUWlr8gvP%2F8KSqSVUPal5qSMIA%2BO%2BOjif4rutvZ7Xn%2B1QeiAA&uploadId=E72208D2BD4C4FE88030B05B52AA5532&x-oss-access-key-id=STS.NSonnwwNjJcE3H7FDEwnjNexU&x-oss-expires=1735025142&x-oss-signature=tTr7dRBtHWCxyMu%2Fq2MwamJKzUDp14xV3OrzPkMc5ps%3D&x-oss-signature-version=OSS2
2024-12-24 06:37:51 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /WEpXPbun%2F889478%2F676a56a1a9b2a4c78cf6430ba2ff65551a6ba83c%2F676a56a18540143e18e9416380c60c6e078d93a0?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=45586F888FF34FC883624AD1A4D8F586&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735025841&x-oss-signature=kiintM%2BqZc9B%2F2d6dvJ0eEbCNQgCBpm82gY5hY8eftA%3D&x-oss-signature-version=OSS2
2024-12-24 06:46:21 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-24 06:46:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-12-24 06:47:38 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /69xNXj1L%2F889478%2F6747b920dd5edf34733b4bfcb42294feb4570508%2F676a58ec8846b222b3f64b799f31983b9a1616da?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=F42B2D0853414AA698992A9787FCDA17&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026428&x-oss-signature=eT5M4gSh6uzIe8hFxZRZpOIaBZyaFAJdMrfgf62pJc4%3D&x-oss-signature-version=OSS2
2024-12-24 06:48:09 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /69xNXj1L%2F889478%2F6747b920dd5edf34733b4bfcb42294feb4570508%2F676a58ec8846b222b3f64b799f31983b9a1616da?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=F42B2D0853414AA698992A9787FCDA17&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026428&x-oss-signature=eT5M4gSh6uzIe8hFxZRZpOIaBZyaFAJdMrfgf62pJc4%3D&x-oss-signature-version=OSS2
2024-12-24 06:48:40 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /69xNXj1L%2F889478%2F6747b920dd5edf34733b4bfcb42294feb4570508%2F676a58ec8846b222b3f64b799f31983b9a1616da?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=F42B2D0853414AA698992A9787FCDA17&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026428&x-oss-signature=eT5M4gSh6uzIe8hFxZRZpOIaBZyaFAJdMrfgf62pJc4%3D&x-oss-signature-version=OSS2
2024-12-24 06:49:56 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-24 06:49:57 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2024-12-24 06:51:59 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/06/16/00/00/57/119671870_p0.png
2024-12-24 06:53:07 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/12/01/18/04/56/124805371_p0.jpg
2024-12-24 06:56:38 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /rQwpU0QB%2F889478%2F66ae26f199e3b6dfb6904049854aed92efe0f50c%2F676a5ae0d516a23e5e50429cbb18caa95c9eb3f3?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=2788E6FD465A4F149D84F9ADEADFFA96&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026928&x-oss-signature=%2BDv9SYkqVej4irx1vkNACKs54FErdH%2Fhzj5m61RQgvg%3D&x-oss-signature-version=OSS2
2024-12-24 06:57:10 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /rQwpU0QB%2F889478%2F66ae26f199e3b6dfb6904049854aed92efe0f50c%2F676a5ae0d516a23e5e50429cbb18caa95c9eb3f3?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=2788E6FD465A4F149D84F9ADEADFFA96&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026928&x-oss-signature=%2BDv9SYkqVej4irx1vkNACKs54FErdH%2Fhzj5m61RQgvg%3D&x-oss-signature-version=OSS2
2024-12-24 06:57:19 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/rQwpU0QB%2F889478%2F66ae26f199e3b6dfb6904049854aed92efe0f50c%2F676a5ae0d516a23e5e50429cbb18caa95c9eb3f3?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=2788E6FD465A4F149D84F9ADEADFFA96&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026928&x-oss-signature=%2BDv9SYkqVej4irx1vkNACKs54FErdH%2Fhzj5m61RQgvg%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A5B4758BFD93232E72646</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"81FEDC636EB3417C39EA571E23116F9E"</PartEtag>
  <PartNumber>2</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 06:57:39 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/rQwpU0QB%2F889478%2F66ae26f199e3b6dfb6904049854aed92efe0f50c%2F676a5ae0d516a23e5e50429cbb18caa95c9eb3f3?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=2788E6FD465A4F149D84F9ADEADFFA96&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026928&x-oss-signature=%2BDv9SYkqVej4irx1vkNACKs54FErdH%2Fhzj5m61RQgvg%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A5B4F61AC2A3731C9B16B</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"81FEDC636EB3417C39EA571E23116F9E"</PartEtag>
  <PartNumber>2</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 06:57:50 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/rQwpU0QB%2F889478%2F66ae26f199e3b6dfb6904049854aed92efe0f50c%2F676a5ae0d516a23e5e50429cbb18caa95c9eb3f3?partNumber=2&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=2788E6FD465A4F149D84F9ADEADFFA96&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735026928&x-oss-signature=%2BDv9SYkqVej4irx1vkNACKs54FErdH%2Fhzj5m61RQgvg%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>676A5B6332187D39330B8E6A</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"81FEDC636EB3417C39EA571E23116F9E"</PartEtag>
  <PartNumber>2</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-12-24 06:57:50 - ERROR - aliyundrive.py - 1077 - File tmp/121132733/121132733_p0.png Part 2 upload failed.
2024-12-24 06:58:33 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /uyDMMoNW%2F889478%2F658ea613e8b6376a53ad4a68b466bfde045846c4%2F676a5b7b08994d4ad54846a68e2ab51bddbaacd0?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=57CD38F91D6841E9B9F9BB0AB05C5ED8&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735027083&x-oss-signature=cdOirdjG1axNnkT5ZDUrMRrYGhESTLFtTbCe8QM%2BctQ%3D&x-oss-signature-version=OSS2
2024-12-24 06:59:41 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /LFFhtV3x%2F889478%2F676a5bbf5cdffc3cba26422eb07b189d0e0909a9%2F676a5bbf25437c32307048bab23ff90c1ff8aac5?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=03FA26CD3F494E7CAF4EC3320E206A11&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735027151&x-oss-signature=wS1DysDA0Stwd6MsOPnE9SRy7ZX9ZaTaLODXds%2FW1EQ%3D&x-oss-signature-version=OSS2
2024-12-24 07:01:18 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /cdiebzIC%2F889478%2F67328705e8fe45b7e9014db8850b41d5f57ec976%2F676a5c20ef05dfd5dfcb4065bb6d9142912239b6?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=A0804DACBE5B41AEBA03AB892C1F9485&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735027248&x-oss-signature=4lZ7wLkYWNvtL2uAeWkpbCc0lATc5pJzj2%2FR3kCoHZY%3D&x-oss-signature-version=OSS2
2024-12-24 07:09:19 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /BDqbo2xu%2F889478%2F676a5dfc2325daf467124148a960b6740655618a%2F676a5dfc8276ee5c89cd490385bd36b2f501f187?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=AC4E2B2D57D442E287F134102685B228&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735027724&x-oss-signature=n0IFZatrQtWeO2BjAXrH%2BCcH%2Fo2qCrWUxPs6pB9OYgk%3D&x-oss-signature-version=OSS2
2024-12-24 07:11:30 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /WmAciDnz%2F889478%2F676a5e83ae4722fc0b124b178b260322964dcc3a%2F676a5e835ea884376a03461391532412df1a1351?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=DA94904D0C2A47408175F629CBA711CC&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735027859&x-oss-signature=g2slt%2BSgyKE9wiMA%2FpPCX1KN5vKe%2FDCfNrH9B5QP2Cw%3D&x-oss-signature-version=OSS2
2024-12-24 07:12:42 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /fHBMVeZQ%2F889478%2F676a5ec49dcb1d56a6234a75a1d73281e0a4ef04%2F676a5ec4ccd081edde5b428cbff51b566cab61bb?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=1653B969B6D54EA09C76524D8EBD8081&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735027924&x-oss-signature=etXL1XIlP72XzKQ%2FPgJpr%2F6wlwNXKnjh4aifHFiNxb4%3D&x-oss-signature-version=OSS2
2024-12-24 07:12:54 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-12-24 07:20:16 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /H0QKulHI%2F889478%2F66bf4ba923880a72989b4e36bd76efce1c4c1674%2F676a6092c6e8dccc8b264444a36676f94277834c?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=DDF61A8475B941FDB624D89C25CDBF3C&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735028386&x-oss-signature=ZGEyQAjDHkArD2C607BhHF0p%2Bv6%2FW2gh7OavTFeJ0hc%3D&x-oss-signature-version=OSS2
2024-12-24 07:20:47 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /H0QKulHI%2F889478%2F66bf4ba923880a72989b4e36bd76efce1c4c1674%2F676a6092c6e8dccc8b264444a36676f94277834c?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=DDF61A8475B941FDB624D89C25CDBF3C&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735028386&x-oss-signature=ZGEyQAjDHkArD2C607BhHF0p%2Bv6%2FW2gh7OavTFeJ0hc%3D&x-oss-signature-version=OSS2
2024-12-24 07:27:14 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /tMYFPZND%2F889478%2F660be76533f56e54bc9640c0960d46fb3618d08a%2F676a62342a5d587da121452790408845eb0f79d8?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=74F796D5A41D4E31AA9A0BD80C76511B&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735028804&x-oss-signature=VrJwOwpqAMLPlk2VMUM6tuQUBg0a1eFjXe1JBODndRU%3D&x-oss-signature-version=OSS2
2024-12-24 07:28:00 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /wZgc6zK8%2F889478%2F671ed231c88c7664e41144b3a5b91bfbbda5ff60%2F676a626224df976bc3454d95b6dbc0286f485ac4?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=B94B2DC2557A4CFF853027CF45746D7F&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735028850&x-oss-signature=0T2wWxtNxtbXmeJy5c%2Fl%2FWHDNRExC3wrX9D5WnTrd4E%3D&x-oss-signature-version=OSS2
2024-12-24 07:40:24 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /HAuXHiaM%2F889478%2F66f4a791270ce3fa667445cba36737a79795cf68%2F676a654afcb1dd812d344bda956354fa754e26da?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=7FAB9271E7D649A9BC8CD76CEC729422&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735029594&x-oss-signature=V43tnlT%2BXs46W6BpFV77HeLlflZzZmCCIF8T34XcplQ%3D&x-oss-signature-version=OSS2
2024-12-24 07:46:47 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /kiWLAi7x%2F889478%2F6729f3ab2745c82c9b804f99ad281da10f79181c%2F676a66c3fbdda7124962448ebefa53923657d8df?partNumber=5&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=3D413DC1E2C24C0EB9C4851AC5E9FE0D&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735029971&x-oss-signature=JEVzwAMzOWGKenUFGVFtWPL4y97OFm7wScC6UtN47%2BE%3D&x-oss-signature-version=OSS2
2024-12-24 07:47:18 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /kiWLAi7x%2F889478%2F6729f3ab2745c82c9b804f99ad281da10f79181c%2F676a66c3fbdda7124962448ebefa53923657d8df?partNumber=5&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=3D413DC1E2C24C0EB9C4851AC5E9FE0D&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735029971&x-oss-signature=JEVzwAMzOWGKenUFGVFtWPL4y97OFm7wScC6UtN47%2BE%3D&x-oss-signature-version=OSS2
2024-12-24 07:48:26 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /19c9c1d4%2F889478%2F6768f575e6bfae56354342ad93c59f7d02795d51%2F676a672c71554c5de7914e50a9e115c70a62ef19?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=F4260CFEB8C24E4C855F32C2ACEDA777&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735030076&x-oss-signature=P2OIXgCdP1hanIH75R0Yb9gOp2tLOrou4jWf1XFRq5w%3D&x-oss-signature-version=OSS2
2024-12-24 07:48:57 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /19c9c1d4%2F889478%2F6768f575e6bfae56354342ad93c59f7d02795d51%2F676a672c71554c5de7914e50a9e115c70a62ef19?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=F4260CFEB8C24E4C855F32C2ACEDA777&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735030076&x-oss-signature=P2OIXgCdP1hanIH75R0Yb9gOp2tLOrou4jWf1XFRq5w%3D&x-oss-signature-version=OSS2
2024-12-24 07:50:53 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /lat5pt1i%2F889478%2F66fc80bc658c672a1104485483c455ad0e78d83c%2F676a67befb71c0c2f8bd48a98c84acb287abc914?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=1C97DCB55D99404BAAC48C0696A27AD0&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735030222&x-oss-signature=rNVd064FrS1G0I3zMSyGwREUeEvxfsiQ4Q62WZVpn2I%3D&x-oss-signature-version=OSS2
2024-12-24 08:02:56 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /4BJn7YLa%2F889478%2F66f49b615c8a97ed4d77483399b9df6b0e218b19%2F676a6a915634bb3d4f1a46de80cd70cc3a373eb7?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=665E48609EF044D0A9D6E1D54E22128B&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735030945&x-oss-signature=ImbZ5lbS%2BXWNqNfBtQxUGazE8FUMTSQsjVX4AhsrrkY%3D&x-oss-signature-version=OSS2
2024-12-24 08:09:53 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /petz77WR%2F889478%2F6768f210b5aa0d834609490cb3fee90e4c8c0cdb%2F676a6c10ab68a59844e64d669fce755863ed83f5?partNumber=4&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=111CFD35D1144310BB86B8CFD3420D18&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031328&x-oss-signature=YcxstTtN0ACttbUNZev8JqXc1pvYUX%2BxQAiWX5PtPzo%3D&x-oss-signature-version=OSS2
2024-12-24 08:10:24 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /petz77WR%2F889478%2F6768f210b5aa0d834609490cb3fee90e4c8c0cdb%2F676a6c10ab68a59844e64d669fce755863ed83f5?partNumber=4&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=111CFD35D1144310BB86B8CFD3420D18&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031328&x-oss-signature=YcxstTtN0ACttbUNZev8JqXc1pvYUX%2BxQAiWX5PtPzo%3D&x-oss-signature-version=OSS2
2024-12-24 08:11:38 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /mejTiOaO%2F889478%2F6768f2399973ac9623be457197901e15d0ae9ea9%2F676a6c9bf25491d5efa244e68cea5505dd4ec865?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=0F186B8172BD44339574F1E9CE0ED837&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031467&x-oss-signature=5vRPEjLHSd%2B9%2FMWlLhbtW5Rvt8um3oGXOACqk2Y0RpA%3D&x-oss-signature-version=OSS2
2024-12-24 08:12:09 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /mejTiOaO%2F889478%2F6768f2399973ac9623be457197901e15d0ae9ea9%2F676a6c9bf25491d5efa244e68cea5505dd4ec865?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=0F186B8172BD44339574F1E9CE0ED837&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031467&x-oss-signature=5vRPEjLHSd%2B9%2FMWlLhbtW5Rvt8um3oGXOACqk2Y0RpA%3D&x-oss-signature-version=OSS2
2024-12-24 08:12:39 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /mejTiOaO%2F889478%2F6768f2399973ac9623be457197901e15d0ae9ea9%2F676a6c9bf25491d5efa244e68cea5505dd4ec865?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=0F186B8172BD44339574F1E9CE0ED837&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031467&x-oss-signature=5vRPEjLHSd%2B9%2FMWlLhbtW5Rvt8um3oGXOACqk2Y0RpA%3D&x-oss-signature-version=OSS2
2024-12-24 08:13:10 - ERROR - base.py - 94 - https://cn-beijing-data.aliyundrive.net/mejTiOaO%2F889478%2F6768f2399973ac9623be457197901e15d0ae9ea9%2F676a6c9bf25491d5efa244e68cea5505dd4ec865?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=0F186B8172BD44339574F1E9CE0ED837&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031467&x-oss-signature=5vRPEjLHSd%2B9%2FMWlLhbtW5Rvt8um3oGXOACqk2Y0RpA%3D&x-oss-signature-version=OSS2:HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Max retries exceeded with url: /mejTiOaO%2F889478%2F6768f2399973ac9623be457197901e15d0ae9ea9%2F676a6c9bf25491d5efa244e68cea5505dd4ec865?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=0F186B8172BD44339574F1E9CE0ED837&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735031467&x-oss-signature=5vRPEjLHSd%2B9%2FMWlLhbtW5Rvt8um3oGXOACqk2Y0RpA%3D&x-oss-signature-version=OSS2 (Caused by ProtocolError('Connection aborted.', TimeoutError('The write operation timed out')))
2024-12-24 08:25:58 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /kpKethJz%2F889478%2F67329c061e8d0a2f64884babb3febe8556b2f945%2F676a6ff86c8494de980540988a43054a9e0a45ec?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=6F3CCA13A47B49E7B8F429B013E943A3&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735032328&x-oss-signature=xSW4rsAH1EkKt1U%2FMaS4%2B8U2IccFt7OMP6QWCehJPdI%3D&x-oss-signature-version=OSS2
2024-12-24 08:35:30 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2024-12-24 08:35:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-12-24 08:47:09 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /478Sk9JQ%2F889478%2F673fb6773b5e3993ee724ca59d33d1bc1a8b486d%2F676a74ad04bacd177db54dd4b0d445ac9dfe00fd?partNumber=3&security-token=CAISvgJ1q6Ft5B2yfSjIr5bvAMKAm4YX%2FLWEO0Xyo2YkPfhWmqzD1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNUOTXwSAq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneQ7xK%2F0Y17uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiSh0QtvXtuMkagAGxpNGKiy4tukaSfsO5hpsq8%2FF1hIUPIBBda%2B%2Bo29AUwz3P7lDq7%2BMYQD9486KGyl2f7UMcMdGi2kDGR8lIYNlimvV8exLi56YjnCJKH6s6pb2M0G20xwYr8Ll9Won3XJiLrmk3icaWjFJi8ZcVM4LrUAma4b2ynNROsulraviOUyAA&uploadId=E42107FFCA054092A9C3584A36835923&x-oss-access-key-id=STS.NUZKx4vY6Kwo9cCGfq1wzuih7&x-oss-expires=1735033534&x-oss-signature=VkhvslTFr81QR3V%2BQ1vUBiQdiK3BIPqldsdoC4M78tY%3D&x-oss-signature-version=OSS2
2024-12-24 08:58:59 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/12/11/21/00/29/125098275_p0.png
